src/components/walk/WalkDefault.tsx

```diff
@@ -0,0 +1,201 @@
+import moment from 'moment';
+import WalkingItemNameButton from '@components/walk/WalkingItemNameButton';
+import { calcRatio } from '@lib/styles/theme';
+import getColorByLevel from '@lib/utils/common/getColorByLevel';
+import { useAppSelector } from '@store/app/hooks';
+import { selectAuth, selectLevel } from '@store/slices/authSlice';
+import {
+  IMoneyRoad,
+  selectisWalkingMoneyRoadsPatched,
+} from '@store/slices/walkingMoneyRoadsSlice';
+import { useEffect, useState } from 'react';
+import styled, { css } from 'styled-components';
+import { ReactComponent as Polygon } from '@assets/icons/walking-selector-polygon.svg';
+import { ReactComponent as D1 } from '@assets/illusts/walk/d-1.svg';
+import { ReactComponent as D2 } from '@assets/illusts/walk/d-2.svg';
+import { ReactComponent as DDay } from '@assets/illusts/walk/d-day.svg';
+import renderItemIllustWalk from '@lib/utils/kid/renderItemIllustWalk';
+import InterestBadge from '@components/common/badges/InterestBadge';
+import SwipeToWalk from '@components/walk/SwipeToWalk';
+import useWalkMoneyRoad from '@lib/hooks/useWalkMoneyRoad';
+import useModals from '@lib/hooks/useModals';
+import Modals, { modals } from '@components/common/modals/Modals';
+
+function WalkDefault({
+  walkingMoneyRoads,
+}: {
+  walkingMoneyRoads: IMoneyRoad[];
+}) {
+  const level = useAppSelector(selectLevel);
+  const patched = useAppSelector(selectisWalkingMoneyRoadsPatched);
+  const { username, isKid, isFemale } = useAppSelector(selectAuth);
+  const colorByLevel = getColorByLevel(level!);
+  const { getWeeklySuccess } = useWalkMoneyRoad(walkingMoneyRoads);
+  const dDayLeft = 7 - moment().day();
+  const [selected, setSelected] = useState<IMoneyRoad>(walkingMoneyRoads[0]);
+  const { openModal, closeModal } = useModals();
+  const { getValue, setValue, getIsAchieved, setIsAchieved } =
+    useWalkMoneyRoad(walkingMoneyRoads);
+
+  const onClickWalkingItemNameButton = (v: IMoneyRoad) => {
+    setSelected(v);
+  };
+
+  useEffect(() => {
+    console.log(patched);
+    if (getWeeklySuccess() && patched) {
+      openModal(modals.primaryModal, {
+        onSubmit: () => {
+          closeModal(modals.primaryModal);
+        },
+        isKid: isKid,
+        isFemale: isFemale,
+        headerText: `${username} 이번주 저금 성공`,
+        bodyText: '뱅키즈와 함께 돈길만 걸어요',
+      });
+    }
+  }, [walkingMoneyRoads]);
+
+  return (
+    <Wrapper>
+      <Header colorByLevel={colorByLevel}>
+        <Title dDayLeft={dDayLeft}>
+          <p>돈길 걷기</p>
+
+          <div>
+            <p>이번주 리셋까지</p>
+            {dDayLeft > 2 && <p>D-{dDayLeft}</p>}
+            {dDayLeft === 2 && <D2 />}
+            {dDayLeft === 1 && <D1 />}
+            {dDayLeft === 0 && <DDay />}
+          </div>
+        </Title>
+        <MoneyRoadList>
+          {walkingMoneyRoads.map((v) => (
+            <div onClick={() => onClickWalkingItemNameButton(v)} key={v.id}>
```

Comment:
```suggestion
            <div onClick={() => onWalkingItemNameButtonClick(v)} key={v.id}>
```
이벤트 핸들러 함수의 네이밍은 동사 + 명사 형태로 합니다.

Reply:
제가 가장 최근에 요청한 PR에 참고자료 링크를 첨부해두었습니다.